web.go

```diff
@@ -23,6 +25,11 @@ var ErrNoHeader = errors.New("jwt: no HTTP Authorization")
 
 var errAuthSchema = errors.New("jwt: want Bearer schema")
 
+type result struct {
+	Status string `json:"status"`
```

Comment:
That "error" label is to comply with <https://tools.ietf.org/html/rfc6750#section-6.2.2>. Is there a reason you'd want to have it in the JSON body?

Reply:
These changes were done as a quick fix for my REST implementation. There are plenty of JWT libraries, but I found that your library is good maintained and quite simple to understand. One of the requirements was, that all endpoints must return valid JSON value, thus I have to modify sources, and thought, that I might not be the only one with these requirements for services.

Reply:
I do like your suggested approach,  
```
        // WriteError allows for custom error responses. When nil it defaults to http.Error.
        // The appropriate WWW-Authenticate header is already present on w.
        WriteError func(w ResponseWriter, error string, code int)

```
because it'll give more freedom, whether somebody wants XML instead of JSON, for example. I'll refactor this.